python/examples/matmul/bench.py

```diff
@@ -144,13 +153,13 @@ def pytorch_kernel(args, sizes, types):
 
 def benchmark():
```

Comment:
Yes we can remove it. Technically it has a slightly different purpose since the run_benchmarks script basically automatically collects all benchmarks that implement a benchmark method. However, I think this has not been used to much so I am fine removing it!